src/main/frontend/components/page.cljs

```diff
@@ -472,7 +472,7 @@
               ;;      (set-setting! :layout value))
               ;;    "graph-layout")]
               [:div.flex.items-center.justify-between.mb-2
-               [:span (t :right-side-bar/journals)]
+               [:span (t :settings-page/enable-journals)]
```

Comment:
This is used in the setting panel of Graph View. So it's better to use `:settings-page/enable-journals` or create a new translate key like `:graph-view/show-journals`